slave.mk

```diff
@@ -9,7 +9,7 @@ SHELL = /bin/bash
 USER = $(shell id -un)
 UID = $(shell id -u)
 GUID = $(shell id -g)
-SONIC_GET_VERSION=$(shell . functions.sh && sonic_get_version)
+SONIC_GET_VERSION=$(shell export BUILD_TIMESTAMP=$(BUILD_TIMESTAMP) && . functions.sh && sonic_get_version)
```

Comment:
can you move the export in sonic_get_version?

Reply:
I did try that, it didn't seem to work.  

Printing  out SONIC_GET_VERSION for debugging:
```
$(info "SONIC_GET_VERSION"                 : "$(SONIC_GET_VERSION)")
```

```
jipan@sonicvm1:~/warm_reboot/sonic-buildimage$ make  target/docker-orchagent-brcm.gz-clean
+++ --- Making target/docker-orchagent-brcm.gz-clean --- +++
BLDENV=stretch make -f Makefile.work stretch
make[1]: Entering directory '/home/jipan/warm_reboot/sonic-buildimage'
SONiC Build System

Build Configuration
"CONFIGURED_PLATFORM"             : "broadcom"
"SONIC_CONFIG_PRINT_DEPENDENCIES" : ""
"SONIC_BUILD_JOBS"                : "1"
"SONIC_CONFIG_MAKE_JOBS"          : "8"
"USERNAME"                        : "admin"
"PASSWORD"                        : "admin"
"ENABLE_DHCP_GRAPH_SERVICE"       : ""
"SHUTDOWN_BGP_ON_START"           : ""
"ENABLE_PFCWD_ON_START"           : ""
"INSTALL_DEBUG_TOOLS"             : ""
"ROUTING_STACK"                   : "frr"
"ENABLE_SYNCD_RPC"                : ""
"ENABLE_ORGANIZATION_EXTENSIONS"  : "y"
"HTTP_PROXY"                      : ""
"HTTPS_PROXY"                     : ""
"ENABLE_SYSTEM_TELEMETRY"         : "y"
"SONIC_DEBUGGING_ON"              : "y"
"SONIC_PROFILING_ON"              : ""
"KERNEL_PROCURE_METHOD"           : "build"
"BUILD_TIMESTAMP"                 : "20181026.123301"
"SONIC_GET_VERSION"                 : "warm-reboot.0-dirty-"
```

With current version:

```
jipan@sonicvm1:~/warm_reboot/sonic-buildimage$ make  target/docker-orchagent-brcm.gz-clean
+++ --- Making target/docker-orchagent-brcm.gz-clean --- +++
BLDENV=stretch make -f Makefile.work stretch
make[1]: Entering directory '/home/jipan/warm_reboot/sonic-buildimage'
SONiC Build System

Build Configuration
"CONFIGURED_PLATFORM"             : "broadcom"
"SONIC_CONFIG_PRINT_DEPENDENCIES" : ""
"SONIC_BUILD_JOBS"                : "1"
"SONIC_CONFIG_MAKE_JOBS"          : "8"
"USERNAME"                        : "admin"
"PASSWORD"                        : "admin"
"ENABLE_DHCP_GRAPH_SERVICE"       : ""
"SHUTDOWN_BGP_ON_START"           : ""
"ENABLE_PFCWD_ON_START"           : ""
"INSTALL_DEBUG_TOOLS"             : ""
"ROUTING_STACK"                   : "frr"
"ENABLE_SYNCD_RPC"                : ""
"ENABLE_ORGANIZATION_EXTENSIONS"  : "y"
"HTTP_PROXY"                      : ""
"HTTPS_PROXY"                     : ""
"ENABLE_SYSTEM_TELEMETRY"         : "y"
"SONIC_DEBUGGING_ON"              : "y"
"SONIC_PROFILING_ON"              : ""
"KERNEL_PROCURE_METHOD"           : "build"
"BUILD_TIMESTAMP"                 : "20181026.123233"
"SONIC_GET_VERSION"                 : "warm-reboot.0-dirty-20181026.123233"
```